### ISSUE TITLE
Spaces in deck overview

### ISSUE THREAD

**Issue by fonol** [krassowski/Anki-Night-Mode#85]
#### Problem description

Hey, love the add-on. What I noticed though is that the deck list on the main page has much bigger space between the entries with nightmode enabled than without.
I checked your styles, and saw you apply 
`height:35px;` to the table columns on the main page. Is there any reason for that, because generally, I like the default, more compact look.

#### Checklist

*Please put x inside brackets if following sentences apply to you:*

 - [x] I verified that I use the lastest add-on version,
 - [x] I verified that I use the lastest Anki version,
 - [x] I tried to disable other add-ons to find the culprit,
 - [x] I checked if anyone reported this problem before, as well as Known issues section on add-ons page,
 - [ ] My issue is solved when holding shift while starting Anki.
 
#### Information about your Anki Setup

<details>

*Please open Anki, go to Help → About, look-up two lines, starting with `Version:` and `Qt` and copy-paste them below:*

```
Version:
Qt:
```

*Please got to Night Mode → About now and copy-paste the first line from the pop-up window below:*

```
Night Mode x.x.x
```

*Please fill in your OS details (Linux/Mac/Windows):*

```
OS: Windows
```
 
 
*If you have a compatibility problem with a specific add-on or add-ons, please paste the add-on page links below:*
 - [Other add-on name](https://)
</details>

#### Backtrace (if any)

<details>

*If you had an error which resulted in producing exception with a backtrace, please paste the backtrace below:*

```python"
[paste your backtrace in here if you got any; otherwise feel free to remove this part]
```

</details>

**Comment by krassowski** [comment]
Fair point, this should be optional. The idea was that this increases readability but I agree - this addon should not force this choice on the users. I am happy to accept a pull request which would make it optional; on the other hand, I would prefer not to simply remove it as some users got used to the larger spacing and would consider this a downgrade.
